File: collaborators/BadrChoubai/multiplyAllNotMe/multiplyAllNotMe.swift
func multiplyAllNotMe(numbers: [Int]) -> [Int] {
  let product = numbers.reduce(1, {x, y in x * y});

  var productArray = Array<Int>()

  for num in numbers {
    productArray.append(product / num)
  }
  return productArray
}

print(multiplyAllNotMe(numbers: [1,2,3,4,5]))

File: collaborators/BadrChoubai/multiplyAllNotMe/multiplyAllNotMe.ts
const multiplyAllNotMe = (numbers: number[]): Array<number> => {
  const product = numbers.reduce((x, y) => x * y);
  let result: number[] = new Array(numbers.length);

  for (let i = 0; i < numbers.length; i++) {
    result[i] = product / numbers[i];
  }
  return result;
}

console.time()
console.log(multiplyAllNotMe([1,2,3,4,5]));
console.timeEnd();